pyproject.toml

```diff
@@ -12,4 +12,4 @@ flake8-implicit-str-concat.allow-multiline=false
 "pytest.main".msg = "pytest.main requires special handling that is easy to mess up!"
 
 [tool.pytest.ini_options]
-addopts = "-n auto"
+addopts = "-n auto --ignore=test_misra.sh"
```

Comment:
we should only consider files that look like `test_*.py`. does this fix it?
```
python_files = ["test_*.py"]
```

Reply:
It didn't work.   `test_*.py` is the default anyway. 

[Shane's issue](https://github.com/pytest-dev/pytest/issues/11974) points to plugins interaction-  `pytest-cpp` in particular. 

We can fix it either with:
- `addopts = "-n auto --ignore-glob='*.sh'"` or
- `cpp_files = "test_*.py"`